### DISCUSSION TITLE
Directly editing the tag field

### DISCUSSION THREAD

**Discussion by Terroosh** [BlafKing/sd-civitai-browser-plus#53] [Ideas]
A lot of models are uploaded without tags, with incomplete tags, or with optional/supplementary tags not included. Because of that, it would be nice if we could type in additional tags in the Trained Tags box.

**Comment by BlafKing** [comment]
Solid idea! that'll also be really easy to implement :)

I'm currently working on a rework of the UI so I'll include that into the update!

**Comment by BlafKing** [comment]
You can now input custom tags into the tag field which will get used in the functions! 👍🏻 

Thanks for the suggestion :)
